cfn/release.yml

```diff
@@ -263,7 +278,7 @@ Resources:
   S3ECReleaseTestS3Bucket:
     Type: 'AWS::S3::Bucket'
     Properties:
-      BucketName: s3ec-release-test-bucket
+      BucketName: !Sub "s3ec-release-test-bucket-${AWS::StackName}"
```

Comment:
The bucket name has been updated because it only stored ephemeral data and having a non-unique name makes testing new changes easier.

Reply:
This change would have upper case char, so we did not push it.